operatorconfig/driverconfig/unity/v2.11.0/driver-config-params.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <DriverDefaultReleaseName>-config-params
+  namespace: <DriverDefaultReleaseNamespace>
+data:
+  driver-config-params.yaml: |
+    CSI_LOG_LEVEL: "info"
```

Comment:
@cbartoszDell could you add an extra parameter to this file after line 8
CSI_LOG_FORMAT: "JSON"
I recently made changes to n, n-1, n-2 version of unity.

Reply:
addressed